daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/ContractDiscriminatorFreshnessCheckSpec.scala

```diff
@@ -135,6 +135,7 @@ class ContractDiscriminatorFreshnessCheckSpec
     engine
       .submit(
         submitters = Set(alice),
+        readAs = Set.empty,
```

Comment:
What does an empty set mean for `readAs`?

Reply:
It means there are no additional readers (besides the submitters).

Reply:
same semantics as on the ledger API

Reply:
Can you point me to the relevant documentation, please?

Reply:
https://github.com/digital-asset/daml/blob/ffc88d52b438dce5703b1b20d7d82f6f3d63337b/ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto#L97